### ISSUE TITLE
[Slack] Configure notifications

### ISSUE THREAD

**Issue by Danetag** [zeit/integrations#75]
Hey team!

I was wondering if it was possible to configure the notifications we are getting in Slack? 

For instance:
- On our current Zeit account, we have 4 different repos. But we only want one of the repo to push notifications in our Slack channel. Right now, any projects within the account notifies the channel
- I'm not sure to understand how frequently it pushes notification. For instance, I created a PR based on a new branch, and after 5 minutes, it pushed 5 similar notifications with the status "CREATED ⚪ " (related to my PR), then 5 notifications with the status "READY ✅ ", while I only made one commit to that PR...
- Can we restrict the notification to certain branches? Ideally, we would push a notification for each new PR,  and every time a few branches are merged (`dev`, `qa`...)

Thank you!

**Comment by nicovalencia** [comment]
Where does the Slack integration code live? Is it open source?

I might be able to contribute, because we will need that functionality fairly soon as well.

**Comment by RenaudROHLINGER** [comment]
@nicovalencia  I believe that the code is here :
https://github.com/vercel/integrations/tree/master/slack

Our company is very interested in restricting the push notifications from a project to a specific channel too. I might try a PR but I have pretty much 0 ideas about how to test the thing.

**Comment by mixn** [comment]
@Danetag Re: the first point you mentioned:

I had the same issue — I needed only two repositories to post to Slack, instead of ALL my repos. And yay 🎉, it’s already possible! 🙂

1. Log into [vercel.com](https://vercel.com/)
2. Go to [vercel.com/dashboard/integrations](https://vercel.com/dashboard/integrations)
3. Pick the (already configured) Slack integration below
4. Click on “Manage Access”
5. Here you’ll be able to restrict the access to (particular) repos of your choice 👏 

<img width="496" alt="1605232756" src="https://user-images.githubusercontent.com/672237/99019410-872ca100-255c-11eb-955f-177a140b54e4.png">
 
Hope that helps with that at least. 🙂

**Comment by homerjam** [comment]
@mixn Unfortunately this doesn't work! All projects send notifications even when a specific project is set

**Comment by cereme** [comment]
@mixn The setting does not work at all here too.

**Comment by bduff9** [comment]
Yes, we need this as well.  The ideal for us would be to select a Slack channel per project as they are already broken out for similar services, i.e. logging, github actions, sentry, etc.  But even if that is not possible, need to be able to limit as we have multiple different projects all sending messages to the same channel even after only selecting the one project that should be sending messages.

**Comment by aksperiod** [comment]
+1 please let me know when this is fixed.
